code/game/machinery/telecomms/broadcasting.dm

```diff
@@ -181,7 +181,7 @@
 
 	// Add observers who have ghost radio enabled.
 	for(var/mob/dead/observer/ghost in GLOB.player_list)
```

Comment:
Null clients should NOT be getting in here. The bug here is not that nulls are not handled, the bug is that there are nulls at ALL.

What is going on

Reply:
Because clients are fickle and dumb

Reply:
If you don't think this is the right solution even though clients-randomly-becoming-null is a very known behavior, then this should at least stack_trace or similar but remove from the list, I know the bug they're talking about and it's disastrous

Reply:
Clients going null for a tick SOMETIMES can happen, particularly in more expansive code inside a for loop like this. That's why we need the checks in like, say code.
Clients going null for a full round, and STAYING null, in this list, implies that logout failed, or we inserted into the list after logout. 
That is very concerning.

I think it's safe to make the non null assertion here, since we have a decent explanation of what causes the random nulls, and this doesn't fit it. Kinda unrelated tho.